qutip/qobj.py

```diff
@@ -978,16 +952,18 @@ def conj(self):
     def norm(self, norm=None, sparse=False, tol=0, maxiter=100000):
         """Norm of a quantum object.
 
-        Default norm is L2-norm for kets and trace-norm for operators.
-        Other ket and operator norms may be specified using the `norm` and
-        argument.
+        Default norm is `L2-norm` for kets and `trace-norm` for operators.
```

Comment:
Why are these back-ticks added?  For one, single backticks just produce italic text (not monospace), and two, L2-norm and trace-norm are just words, so don't need any formatting?